MetaMask/metamask-mobile
# Default account icons "Jazzicons" to match the same settings of extension

**Description**

Changing the redux initial state value of icons to be the jazz icon type to match the extension initial state instead of blockie icon type.

Icon types example:
![image](https://user-images.githubusercontent.com/46944231/165582546-2a67ade6-1c9b-432e-84a6-03f2a22be7e6.png)

Test cases:
 
Case 1:
* Open the Metamask app on the device
* Go to Settings -> General
* Change icon type (remember what icon type you changed for)
* Kill the app
* Open the app
* The icon type should be the one that was selected
 
Case 2:
* Delete the MetaMask app from the device
* Install it again
* Import one existing account
* The icons should be jazzIcons 
 
Case 3:
* Delete the MetaMask app from the device 
* Install it again
* Create a new account
* The icons should be jazzicons

**Checklist**


**Issue**

Progresses #4158
